erizo_controller/erizoClient/src/Stream.js

```diff
@@ -280,25 +280,41 @@ Erizo.Stream = function (spec) {
         }
     };
 
+    that.muteAudio = function (audioActive, callback) {
```

Comment:
It may take a while before we have video mute (if we ever implement it). I wanted to avoid the confusion in ErizoClient API while leaving the rest of the path flexible enough so we don't have to change many things in the future.
I still prefer muteAudio but I could be convinced to change it :)  @jcague, any opinion on this?